packages/medusa/src/repositories/publishable-api-key-sales-channel.ts

```diff
@@ -8,11 +8,13 @@ export class PublishableApiKeySalesChannelRepository extends Repository<Publisha
    * Query a list of sales channels that are assigned to the publishable key scope
    *
    * @param publishableApiKeyId - id of the key to retrieve channels for
+   * @param q - free text search param
    */
   public async findSalesChannels(
-    publishableApiKeyId: string
+    publishableApiKeyId: string,
+    q?: string
```

Comment:
suggestion (non-blocking): make this a query object instead so we can add more filters in the future if needed